isb/Dockerfile

```diff
@@ -47,7 +47,7 @@ COPY ./isamples_webui/index.html /app/isb_web/ui/
 COPY ./isamples_webui/manifest.json /app/isb_web/ui/
 COPY ./isamples_webui/robots.txt /app/isb_web/ui/
 COPY ./isamples_webui/static /app/isb_web/ui/static
-RUN echo "const _solr_base = \"$ISB_SITEMAP_PREFIX/thing/\";\n\nconst config = {\n  \"solr_url\": _solr_base + \"select\",\n  \"solr_stream\": _solr_base + \"stream\",\n  \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n  \"analytics_domain\": \"$ANALYTICS_DOMAIN\",\n  \"datacite_prefix\": \"$DATACITE_PREFIX\",\n  \"datacite_publisher\": \"$DATACITE_PUBLISHER\"\n};\n\nwindow.config = config;" | tee /app/isb_web/ui/config.js
+RUN echo "const _server_base = \"$ISB_SITEMAP_PREFIX/\";\n\nconst orcid_client_id = \"APP-4LPIZXDJ25K3MV8C\";\nconst orcid_response_type = \"code\";\nconst orcid_redirect_uri = \"http://localhost:3000/orcid_token\";\nconst orcid_scope = \"/authenticate\";\nconst orcid_endpoint = \"https://sandbox.orcid.org/oauth/authorize\";\n\nconst isamples_orcid_token = \"http://localhost:8000/orcid_token\";\nconst DATACITE_PUBLISHER = [\'Sesar\', \'Geome\', \"OpenContext\", \"Smithsonian\"];\n\n\nconst config = {\n  \"solr_url\": _server_base + \"thing/select\",\n  \"solr_stream\": _server_base + \"thing/stream\",\n  \"thingpage\": _server_base + \"thingpage\",\n  \"dois_draft\": _server_base + \"mint_draft_identifiers\",\n  \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n  \"analytics_domain\": \"isamples.org\",\n  \"datacite_prefix\": \"10.82273\",\n  \"datacite_publisher\": DATACITE_PUBLISHER,\n  \"orcid_auth\": `${orcid_endpoint}?client_id=${orcid_client_id}&response_type=${orcid_response_type}&scope=${orcid_scope}&redirect_uri=${orcid_redirect_uri}`,\n  \"ordic_token\": isamples_orcid_token,\n\n};\n\nwindow.config = config;" | tee /app/isb_web/ui/config.js
```

Comment:
We shouldn't be checking anything with `localhost` into this repository.  This is the repository with code that we use to build our production containers.  Anything that has `localhost` in it is guaranteed not to work.